substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/heap/PhysicalMemory.java

```diff
@@ -106,6 +111,7 @@ public static UnsignedWord size() {
      * Returns the size of physical memory in bytes that has been previously cached. This method
      * must not be called if {@link #isInitialized()} is still false.
      */
+    @Uninterruptible(reason = "Called from uninterruptible code.", mayBeInlined = true)
```

Comment:
This came from https://github.com/oracle/graal/pull/7371 in master, so we should either backport that one separately or remove it? Either way not part of the original changeset.

Reply:
Removed.